src/components/structures/TagPanel.js

```diff
@@ -50,6 +50,8 @@ const TagTile = React.createClass({
         dis.dispatch({
             action: 'select_tag',
             tag: this.props.groupProfile.groupId,
+            ctrlOrCmdKey: e.metaKey || e.ctrlKey,
+            shiftKey: e.shiftKey,
```

Comment:
Hmm, so:

1. This will also catch the windows key on windows and the ctrl key on mac, which isn't quite what you want. Fortunately we've already written what you want in https://github.com/matrix-org/matrix-react-sdk/blob/master/src/components/structures/LoggedInView.js#L156 (which should totally be factored out... ;) )

2. I'm not really keen on passing the state of the modifier keys here: I'd have thought it should be up to the thing presenting the UI to decide what the modifier keys do.